src/neptune/metadata_containers/run.py

```diff
@@ -309,7 +311,7 @@ def __init__(
         project: Optional[str] = None,
         api_token: Optional[str] = None,
         custom_run_id: Optional[str] = None,
-        mode: Optional[str] = None,
+        mode: Optional[Literal["async", "sync", "offline", "read-only", "debug"]] = None,
```

Comment:
It's never been supported except for `Runs`

Reply:
You can take a look at: https://github.com/neptune-ai/neptune-client/blob/15cb94dcdc4a3b8f45923097207b7523e52bec02/src/neptune/metadata_containers/project.py#L155 . There is an exception about it. Probably we should adjust docs as well.

Reply:
Thankfully it's correct in the docs, I just had a mind glitch. (We don't explicitly say "for runs only" where offline mode is described, so it may not hurt to clarify it there.)